ui/app/components/pages/first-time-flow/create-password/import-with-seed-phrase/import-with-seed-phrase.component.js

```diff
@@ -45,9 +45,11 @@ export default class ImportWithSeedPhrase extends PureComponent {
       } else if (!validateMnemonic(parsedSeedPhrase)) {
         seedPhraseError = this.context.t('invalidSeedPhrase')
       }
+
```

Comment:
Test failures are real failures. `parsedSeedPhrase` will always remove trailing white spaces, not allowing the user to type spaced words.

Reply:
Yeah totally didn't test this